m2c/asm_file.py

```diff
@@ -339,7 +339,9 @@ def re_comment_replacer(match: Match[str]) -> str:
     re_whitespace_or_string = re.compile(r'\s+|"(?:\\.|[^\\"])*"')
     re_local_glabel = re.compile("L(_.*_)?[0-9A-F]{8}")
     re_local_label = re.compile("loc_|locret_|def_|lbl_|LAB_|jump_")
-    re_label = re.compile(r'(?:([a-zA-Z0-9_.$]+)|"([a-zA-Z0-9_.$<>@,-]+)"):')
+    re_label = re.compile(
+        r'(?:(?:\.obj )?([a-zA-Z0-9_.$]+)|"([a-zA-Z0-9_.$<>@,-]+)")[:,]'
```

Comment:
Instead of complicating this regex, can you add a case near where jlabel/glabel/etc are handled?

Reply:
I will give it a try.